### DISCUSSION TITLE
Sqlite3 module not found error on another machine

### DISCUSSION THREAD

**Discussion by LesHol** [brentvollebregt/auto-py-to-exe#481] [Q&A]
Hi
Auto-py-to-exe is fantastic but I have a strange problem with sqlite3. When I run Auto-py... it finishes without any errors and I can run the app with no problem but when I try to run it on another machine I get "module not found error". I have read your FAQ and tried a lot of different ways to get it working. I wrote a small app which just included the import and a print statement and then it worked on the other machine.

Please help me , I have been trying for weeks now!

I can mail you the DEBUG file, it is huge! I find a couple of refrences to Sqlite3 which seems Ok and no warnings.

[DEBUG running auto.txt](https://github.com/brentvollebregt/auto-py-to-exe/files/15209279/DEBUG.running.auto.txt)

The error message:

Traceback (most recent call last):
  File "main.py", line 2, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "sqlite3\__init__.py", line 57, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "sqlite3\dbapi2.py", line 27, in <module>
ImportError: DLL load failed while importing _sqlite3: Det går inte att hitta den angivna modulen.

**Comment by brentvollebregt** [comment]
If you package to "One Directory" and then look in the `_internal` folder output, does it contain `_sqlite3.pyd`? This is the missing file I believe and it exists for me when I import `sqlite3`.

**Reply by LesHol** [reply]
Thank you for your fast response.

Yes the sqlite3.pyd is there. The strange thing is when I move all of this to another machine I get this error but not on my own machine. Something must be missing I believe.

Regards

**Reply by brentvollebregt** [reply]
If you package just the following, do you get the same results?

```py
import sqlite3
print('working')
```

When executing the exe from cmd, you should see "working" printed.

**Reply by LesHol** [reply]
That works without problem! I forgot to tell you that sqlite3 is a compiled version with Sqlcipher4 but it shouldn`t make any difference. I shall now try with another app which have some database instructions in it and see if that works. Now I have tried this to and get the same error but when running it on my own machine, no problem!
Maybe there is something missing. Have you got any suggestions of any parameters I can use in Auto-py?

**Reply by brentvollebregt** [reply]
I'm not aware of what files are required for Sqlcipher4.

I recommend creating a minimal reproducible example to identify where the error truly lies. If it is in Sqlcipher4, you might need to hit them up - there is nothing I can do sorry. It sounds like specific files are not being included but it is hard to know what is missing.

**Reply by LesHol** [reply]
I understand. Thank you for your help.
